app/pods/bucket-type/route.js

```diff
@@ -16,7 +16,13 @@ export default Ember.Route.extend({
 
     setupController: function(controller, model) {
         this._super(controller, model);
-
+        if(!model.get('bucketList')) {
+            // Init to empty bucket list
+            let buckets = null;
+            let emptyList = this.explorer.createBucketList(buckets, model.get('cluster'),
+                model.get('bucketType'), this.store);
+            model.set('bucketList', emptyList);
+        }
```

Comment:
Does it make sense to have this logic here versus adding it to the `explorer. getBucketTypeWithBucketList` method?